internal/providers/terraform/azure/testdata/cosmosdb_cassandra_keyspace_test/cosmosdb_cassandra_keyspace_test.golden

```diff
@@ -0,0 +1,72 @@
+
+ Name                                                 Monthly Qty  Unit                Monthly Cost 
+                                                                                                    
+ azurerm_cosmosdb_cassandra_keyspace.autoscale                                                      
+ ├─ Request units (provisioned, West US)                   43,800  hours                    $350.40 
+ ├─ Request units (provisioned, Central US)                43,800  hours                    $350.40 
+ ├─ Request units (provisioned, Japan West)                43,800  hours                    $394.20 
+ ├─ Transactional storage (West US)               Cost depends on usage: $0.25 per GB               
+ ├─ Analytical Storage (West US)                  Cost depends on usage: $0.03 per GB               
+ ├─ Write operations (West US)                    Cost depends on usage: $0.06 per 10K operations   
+ ├─ Read operations (West US)                     Cost depends on usage: $0.0054 per 10K operations 
+ ├─ Transactional storage (Central US)            Cost depends on usage: $0.25 per GB               
+ ├─ Analytical Storage (Central US)               Cost depends on usage: $0.02 per GB               
+ ├─ Write operations (Central US)                 Cost depends on usage: $0.05 per 10K operations   
+ ├─ Read operations (Central US)                  Cost depends on usage: $0.005 per 10K operations  
+ ├─ Transactional storage (Japan West)            Cost depends on usage: $0.28 per GB               
+ ├─ Analytical Storage (Japan West)               Cost depends on usage: $0.03 per GB               
+ ├─ Write operations (Japan West)                 Cost depends on usage: $0.05 per 10K operations   
+ ├─ Read operations (Japan West)                  Cost depends on usage: $0.005 per 10K operations  
+ ├─ Pereodic backup (West US)                     Cost depends on usage: $0.15 per GB               
+ ├─ Pereodic backup (Central US)                  Cost depends on usage: $0.15 per GB               
+ ├─ Pereodic backup (Japan West)                  Cost depends on usage: $0.16 per GB               
+ └─ Restored data                                 Cost depends on usage: $0.15 per GB               
+                                                                                                    
+ azurerm_cosmosdb_cassandra_keyspace.provisioned                                                    
+ ├─ Request units (provisioned, West US)                    3,650  hours                     $29.20 
```

Comment:
For provisioned I think the displayed unit  should be "RU/s x 100" and then set `UnitMultiplier: schema.HourToMonthUnitMultiplier,` so it displays as
```
├─ Request units (provisioned, West US)                    500  RU/s x 100                     $29.20 
```
@alikhajeh1 thoughts?

Reply:
+1, that seems clearer to understand as it maps to the pricing webpage.